{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -23,37 +23,52 @@
 
 
 def running_from_cluster() -> bool:
+
```

Comment:
I suggest you pull https://github.com/ihmeuw/vivarium_cluster_tools/blob/main/src/vivarium_cluster_tools/psimulate/cluster/interface.py#L17-L23 up to the top level of vct and use it here instead.